### ISSUE TITLE
Improve accessibility test failure messages

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#1540]
Whenever one of the main Cypress tests fails, the error message usually provides a good hint as to what went wrong.

But when an accessibility one fails, it's completely opaque, usually something like "[2 accessibility violations were detected: expected 2 to equal 0](https://github.com/galaxyproject/galaxy-hub/runs/7367243282?check_suite_focus=true#step:9:92)".

To get any clue, you have to run `cypress:open`, then run the a11y test, click on the "- a11y error!"s, then pick the actual, informative error messages out of the terminal output.

Getting these messages to print out in the log messages accessible through Github might be too difficult. But it is an issue I keep running into. So I figured I'd note it here, and in the process document the current steps to find the full error messages.
